### DISCUSSION TITLE
Creeper Overhaul for Creeper Quest Compatibility

### DISCUSSION THREAD

**Discussion by ardissaps** [ejektaflex/Bountiful#194] [Mod Compat]
Would you please include bounty for other mods creeper to creeper quest list? (like kill any type of creeper instead of only vanilla creeper)
https://www.curseforge.com/minecraft/mc-mods/creeper-overhaul

**Comment by ejektaflex** [comment]
I'll have to look into this - if I remember correctly, it depends on how they created their creepers.
